tsakhloev/second/task1.py

```diff
@@ -0,0 +1,73 @@
+"""
+По введеній користувачем даті вивести день тиждня
+"""
+
+from re import match
+
+def check_input( day, month, year ):
+
+    if year >= 1 and month <= 12 and month >= 1 and day >= 1:
```

Comment:
In order to shorten the string, It wil be better to use  "if all( [year >= 1, month <= 12, month >= 1, day >= 1])"